### PR TITLE
doc: Update mar-2022-security-releases.md

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
...
 layout: blog-post.hbs
 author: Joe Sepi
 ---
 
+# Summary
+
+The Node.js project will release new versions of the 12.x, 14.x, 16.x, and 17.x
+releases lines on or shortly after Thursday, March 17th, 2022 to incorporate 
+upstream patches from OpenSSL.
+
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md:15]:
```suggestion

## Impact

The 17.x release line of Node.js is vulnerable to one High severity issue.

The 16.x release line of Node.js is vulnerable to one High severity issue.

The 14.x release line of Node.js is vulnerable to one High severity issue.

The 12.x release line of Node.js is vulnerable to one High severity issue.
```

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
 ---
 date: 2022-03-14T12:00:00.000Z
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
date: 2022-03-16T23:22:00.000Z
```

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+Please follow the process outlined in https://github.com/nodejs/node/blob/master/SECURITY.md
+if you wish to report a vulnerability in Node.js.
+
+Subscribe to the low-volume announcement-only nodejs-sec mailing list at https://groups.google.com/forum/#!forum/nodejs-sec to stay up to date on security vulnerabilities and security-related releases of Node.js and the projects maintained in the nodejs GitHub organization.
+
+
+---
+
+(previous content)
+
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
```

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 4]
```diff
@@ -1,12 +1,36 @@
...
 date: 2022-03-14T12:00:00.000Z
 category: vulnerability
-title: OpenSSL security releases may require Node.js security releases
+title: OpenSSL security releases require Node.js security releases
 slug: openssl-and-high-severity-fixes-mar-2022
 layout: blog-post.hbs
 author: Joe Sepi
 ---
 
+# Summary
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
# _(Update 16-Jan-2022)_ Summary
```

Comment (richardlau) [reply locale/en/blog/vulnerability/mar-2022-security-releases.md]: March?

**Review by mhdawson — APPROVED** [review]
LGTM

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 5]
```diff
@@ -1,12 +1,36 @@
...
+
+The Node.js project will release new versions of the 12.x, 14.x, 16.x, and 17.x
+releases lines on or shortly after Thursday, March 17th, 2022 to incorporate
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
releases lines on or shortly after Thursday, March 17th, 2022 to incorporate
```

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 6]
```diff
@@ -1,12 +1,42 @@
...
+
+## Contact and future updates
+
+The current Node.js security policy can be found at https://github.com/nodejs/node/blob/master/SECURITY.md.
+Please follow the process outlined in https://github.com/nodejs/node/blob/master/SECURITY.md
+if you wish to report a vulnerability in Node.js.
+
+Subscribe to the low-volume announcement-only nodejs-sec mailing list at https://groups.google.com/forum/#!forum/nodejs-sec to stay up to date on security vulnerabilities and security-related releases of Node.js and the projects maintained in the nodejs GitHub organization.
+
+
```
Comment (mhdawson) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
```

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 7]
```diff
@@ -1,12 +1,41 @@
...
+## Impact
+
+The 17.x release line of Node.js is vulnerable to one High severity issue.
+
+The 16.x release line of Node.js is vulnerable to one High severity issue.
+
+The 14.x release line of Node.js is vulnerable to one High severity issue.
+
+The 12.x release line of Node.js is vulnerable to one High severity issue.
+## Release timing
```
Comment (richardlau) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md:26]:
```suggestion

## Release timing
```

**Review by richardlau — APPROVED** [review]

**Pull request merged**